zarr/storage.py

```diff
@@ -76,7 +76,7 @@
     default_compressor = Zlib()
 
 
-Path = Union[str, bytes, None]
+PathType = Union[str, bytes, None]
```

Comment:
This is probably the only change that I don't feel competent to individually review.

Does anyone know the possible impact of this? In my experience, this would count as a breaking change.

Reply:
This could be reverted. From its usage it looked pretty internal (and only in the one module) but I suppose someone downstream could be using it.

Reply:
> but I suppose someone downstream could be using it.

*nods* That was my thought. I have _zero_ evidence that someone is actually doing that, but just in terms of the semvers'ioning